invenio_records_resources/resources/errors.py

```diff
@@ -70,6 +74,36 @@ def __init__(self, error):
         super().__init__(code=500, description=_("Internal server error"))
 
 
+class HTTPJSONException(_HTTPJSONException):
+    """HTTPJSONException that supports setting some extra body fields."""
+
+    def __init__(self, code=None, errors=None, **kwargs):
+        """Constructor."""
+        description = kwargs.pop("description", None)
+        response = kwargs.pop("response", None)
+        super().__init__(code, errors, description=description, response=response)
+        self._extra_fields = kwargs
+
+    def get_body(self, environ=None, scope=None):
+        """Get the response body."""
+        body = {
+            "status": self.code,
+            "message": self.get_description(environ),
+            **self._extra_fields,
+        }
+
+        errors = self.get_errors()
+        if errors:
+            body["errors"] = errors
+
+        # TODO: Revisit how to integrate error monitoring services. See issue #56
```

Comment:
What is the issue #56?
It is not easy to understand well what is the issue here.

Reply:
Right! Thank you for pointing this out. This code was moved here from [invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/pull/1490/files#diff-4e526346ebe67a7c163c6811b36f9b13a88fbc8058ba43b0b1fba00e89fa297eL50). And apparently that was copied from flask-resources, I have updated the comment to point to the correct [issue](https://github.com/inveniosoftware/flask-resources/issues/56)